spec/filters/i18n/frontmatter_filter_spec.rb

```diff
@@ -5,22 +5,75 @@
     <<~FRONTMATTER
       ---
       title: Overview
-      meta_title: Send messages via SMS, WhatsApp, Viber and Facebook Messenger
+      meta_title: Send messages via: SMS, WhatsApp, Viber and Facebook Messenger
```

Comment:
Why is the `:` important?

Reply:
just to make sure that we escape just the key. Otherwise, it might get greedy and consider the key as everything up to the last `:`

Reply:
Ah, that is reasonable. Definitely helps ensure the test does what we want it to do.